cdap-ui/app/cdap/components/AbstractWidget/SqlSelectorWidget/FieldRow.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import TableCell from '@material-ui/core/TableCell';
+import Checkbox from '@material-ui/core/Checkbox';
+import Typography from '@material-ui/core/Typography';
+
+import { IFieldSchema } from 'components/AbstractWidget/SqlSelectorWidget';
+
+import Input from 'components/AbstractWidget/DefaultInput';
+
+interface IFieldRowProps {
+  onFieldChange: (arg0: any) => void;
```

Comment:
Here as well. Lets try avoid using `any` as much as we can. If we can represent that as type then it gives us much more clarity on what it is and ts helps in prevent any unintended changes in the future.